src/wp-includes/class-wp-theme-json.php

```diff
@@ -2646,8 +2645,8 @@ public function get_root_layout_rules( $selector, $block_metadata ) {
 			$content_size = static::is_safe_css_declaration( 'max-width', $content_size ) ? $content_size : 'initial';
 			$wide_size    = isset( $settings['layout']['wideSize'] ) ? $settings['layout']['wideSize'] : $settings['layout']['contentSize'];
 			$wide_size    = static::is_safe_css_declaration( 'max-width', $wide_size ) ? $wide_size : 'initial';
-			$css .= static::ROOT_CSS_PROPERTIES_SELECTOR . ' { --wp--style--global--content-size: ' . $content_size . ';';
-			$css .= '--wp--style--global--wide-size: ' . $wide_size . '; }';
+			$css         .= static::ROOT_CSS_PROPERTIES_SELECTOR . ' { --wp--style--global--content-size: ' . $content_size . ';';
+			$css         .= '--wp--style--global--wide-size: ' . $wide_size . '; }';
```

Comment:
Hmm this change popped up in #6656 too; pretty sure it's an auto-lint thing but I'm not 100% sure if it's correct? Should `.=` be aligned with `=`? Cc @mukeshpanchal27 who usually knows these things 😄

Reply:
Yes alignment looks good.

Reply:
Not sure if it helps, but it looks like in other parts of the codebase, the `.` is one character to the left of the `=` so that the `=` is always in the same column:

https://github.com/WordPress/wordpress-develop/blob/a1a7548a4b584997c00dd5b8e170fe5480144a75/src/wp-admin/edit-form-advanced.php#L292-L293

https://github.com/WordPress/wordpress-develop/blob/f058bd12170560976365239279b532536acd17a3/src/wp-admin/nav-menus.php#L723-L724

Reply:
Awesome, thanks for confirming!